docs/querying/datasource.md

```diff
@@ -272,7 +272,7 @@ GROUP BY
 Join datasources allow you to do a SQL-style join of two datasources. Stacking joins on top of each other allows
 you to join arbitrarily many datasources.
 
-In Druid {{DRUIDVERSION}}, joins are implemented with a broadcast hash-join algorithm. This means that all tables
+In Druid {{DRUIDVERSION}}, joins are implemented with a broadcast hash-join algorithm. This means that all datasources
 other than the leftmost "base" table must fit in memory. It also means that the join condition must be an equality. This
```

Comment:
```suggestion
other than the leftmost "base" datasource must fit in memory. It also means that the join condition must be an equality. This
```
@benkrug should this remain `datasource` throughout the entire paragraph?

Reply:
@techdocsmith thanks for looking at this!  You're right, line 276 should also use 'datasource' rather than 'table'.  I forgot that neither side of a join needs to be a table datasource.  (I only caught the first because the right-hand side *can't* be a table.  But the left-hand side doesn't need to be one, either.)